web/src/loki-client.ts

```diff
@@ -126,6 +137,34 @@ export const executeQueryRange = ({
   );
 };
 
+export const executeVolumeRange = ({
+  query,
+  start,
+  end,
+  config,
+  tenant,
+  namespace,
+}: VolumeRangeParams): CancellableFetch<VolumeRangeResponse> => {
+  const extendedQuery = queryWithNamespace({
+    query,
+    namespace,
+  });
+
+  const params: Record<string, string> = {
+    query: extendedQuery,
+    start: String(start * 1000000),
+    end: String(end * 1000000),
+    limit: String(config?.logsLimit ?? 100),
```

Comment:
Do we need this limit parameter for the volume api?

Reply:
There is a limit defined in the [Loki docs](https://grafana.com/docs/loki/next/reference/loki-http-api/#query-log-volume:~:text=parameter%20is%20required.-,limit,-%3A%20How%20many%20metric), so I just put it to the logs limit default. Realistically though since the volume API is generally very fast we might not need one, but I think we should keep the limit since it's in the Loki docs.

Reply:
But then is not the same as the logsLimit, as this is intended for the logs table. The limit here is the number of metric series to be returned, if the default is 100 we could skip it.